### DISCUSSION TITLE
Make the nozzle not end the purge line when over the initial purged blob

### DISCUSSION THREAD

**Discussion by HitLuca** [jschuh/klipper-macros#197] [Ideas]
I am not 100% sure if this is already fixed, as I'm using an older version of this code, but I noticed a problem that comes up every now and then when printing: the nozzle produces an initial purge blob, then draws a purge line going back and forth. When done, the nozzle moves to the actual print area, but since it always ends the purge line on the purge blob it sometimes brings the purged filament with it. 
This happens on larger prints, where I believe the time it takes to finish the purge line is enough to have the blob solidify and get it tangled with the nozzle.

What I suggest is to have the nozzle end the purge sequence on the other extreme of the purge line, where there is no blob of filament to pull.

**Comment by jschuh** [comment]
Duplicate of #184
